netutils/ftpc/ftpc_connect.c

```diff
@@ -152,8 +148,8 @@ int ftpc_reconnect(FAR struct ftpc_session_s *session)
 
   /* Set up a timer to prevent hangs */
 
-  ret = wd_start(session->wdog,
-                 session->conntimeo, ftpc_timeout, 1, session);
+  ret = wd_start(&session->wdog, session->conntimeo,
```

Comment:
Hmm.. this is another violation of the portable POSIX OS interface.  wd_start(0 is not an application accessible interface.  It is only for use inside the OS.